include/deal.II/base/mpi.h

```diff
@@ -708,6 +708,44 @@ namespace Utilities
            const T &          object_to_send,
            const unsigned int root_process = 0);
 
+    /**
+     * Given a partitioned index set space, compute the owning MPI process rank
+     * of each element of a second index set according to the partitioned index
+     * set. A natural usage of this function is to compute for each ghosted
+     * degree of freedom the MPI rank of the process owning that index.
+     *
+     * The user is responsible to partition the input index set space correctly.
```

Comment:
what does "correctly" here mean? I think you want to say something like this:
```suggestion
     * The first argument passed to this function, @p owned_dofs, must uniquely partition
     * an index space between all processes.
```